### DISCUSSION TITLE
Add support for Cloudflare Turnstyle

### DISCUSSION THREAD

**Discussion by gOOvER** [pelican-dev/panel#515] [Ideas]
I would be great, when you could add support for Cloudflare Turnstyle. In my eyes, this Catcha is better than google.

More infos: https://developers.cloudflare.com/turnstile/

**Comment by rmartinoscar** [comment]
Hey setting this in .env should do the trick (untested)
```
RECAPTCHA_DOMAIN="https://challenges.cloudflare.com/turnstile/v0/siteverify"
```

**Comment by Boy132** [comment]
See https://github.com/pelican-dev/panel/discussions/51
